core/expressionParser/helpers/evaluator.js

```diff
@@ -19,6 +19,7 @@ const stdlib = {
   // arity 1+ (arity 0 allowed by JS)
   min: [Math.min, 1, -1],
   max: [Math.max, 1, -1],
+  includes: [(items, value) => items && items instanceof Array && items.includes(String(value)), 2, 2],
```

Comment:
should use `Array.isArray(items)` replacing ` items && items instanceof Array`

Reply:
yes, thanks, you are right